KoreBuild-dotnet/build/KoreBuild.cmd

```diff
@@ -0,0 +1,31 @@
+@ECHO off
```

Comment:
Don't fully understand - why do we need this file? Why not just do all of this work in build.cmd (it'll have to pull down Sake, but that doesn't seem super problematic?)